service-api/module/Application/src/Controller/IdentityController.php

```diff
@@ -35,8 +37,29 @@ public function indexAction(): JsonModel
         return new JsonModel();
     }
 
-    public function createAction(): void
+    public function createAction(): JsonModel
     {
+        $data = json_decode($this->getRequest()->getContent(), true);
+
+        $caseData = CaseData::fromArray($data);
+
+        if ($caseData->isValid()) {
+            $uuid = Uuid::uuid4();
+            $item = [
+                'id'            => ['S' => $uuid->toString()],
+                'personType'     => ['S' => $data["personType"]],
+                'firstName'     => ['S' => $data["firstName"]],
+                'lastName'      => ['S' => $data["lastName"]],
+                'dob'           => ['S' => $data["dob"]],
+                'verifyMethod'  => ['S' => $data["verifyMethod"]],
+                'lpas'          => ['SS' => $data['lpas']]
+            ];
+
+            $this->dataImportHandler->insertData('cases', $item);
+
+            return new JsonModel(['uuid' => $uuid]);
+        }
+        return new JsonModel(['error' => 'Invalid data']);
```

Comment:
Should set the response status to 400 to indicate that the requested failed and it's the client's fault